README.md

```diff
@@ -48,11 +48,11 @@ By using [Packer's DigitalOcean Builder](https://www.packer.io/docs/builders/dig
 
 This configuration uses Packer's [file provisioner](https://www.packer.io/docs/provisioners/file.html) to upload complete directories to the Droplet. The contents of `files/var/` will be uploaded to `/var/`. Likewise, the contents of `files/etc/` will be uploaded to `/etc/`. One important thing to note about the file provisioner, from Packer's docs:
 
-> The destination directory must already exist. If you need to create it, use a shell provisioner just prior to the file provisioner in order to create the directory. If the destination directory does not exist, the file provisioner may succeed, but it will have undefined results.
+> The destination directory must already exist. If you need to create it, use a shell provisioner just before the file provisioner to create the directory. If the destination directory does not exist, the file provisioner may succeed, but it will have undefined results.
 
 This configuration also uses Packer's [shell provisioner](https://www.packer.io/docs/provisioners/shell.html) to run scripts from the `/scripts` directory and update installed APT packages using an inline task.
 
-After making changes to the configuration, packer validate command.
+After making changes to the configuration, the packer validates the command.
```

Comment:
I think this may be slightly clearer: `After making changes to the configuration, run the packer validate command.`

Reply:
I got it wrong maybe! Should I change this to this `After making changes to the configuration, run the packer validate command.`?

Reply:
Sure, sounds good to me.